server/protocols/jmap-draft-integration-testing/jmap-draft-integration-testing-common/src/test/java/org/apache/james/jmap/draft/methods/integration/GetMailboxesMethodTest.java

```diff
@@ -681,6 +681,28 @@ public void getMailboxesShouldReturnSharedMailboxesWithRead() throws Exception {
             .body(ARGUMENTS + ".list.id", hasItems(mailboxId.serialize()));
     }
 
+    @Test
+    public void getMailboxesShouldReturnSharedMailboxesWithoutRole() throws Exception {
+        MailboxPath bobMailboxPath = MailboxPath.forUser(BOB, DefaultMailboxes.INBOX);
+        MailboxId mailboxId = mailboxProbe.createMailbox(bobMailboxPath);
+
+        mailboxProbe.createMailbox(MailboxConstants.USER_NAMESPACE, ALICE.asString(), DefaultMailboxes.INBOX);
+
+        aclProbe.replaceRights(bobMailboxPath, ALICE.asString(), new Rfc4314Rights(Right.Lookup));
+
+        given()
+            .header("Authorization", accessToken.asString())
+            .body("[[\"getMailboxes\", {\"ids\": [\"" + mailboxId.serialize() + "\"]}, \"#0\"]]")
+        .when()
+            .post("/jmap")
+        .then()
+            .statusCode(200)
+            .body(NAME, equalTo("mailboxes"))
+            .body(ARGUMENTS + ".list", hasSize(1))
+            .body(FIRST_MAILBOX + ".role", nullValue())
```

Comment:
From @mbaechler in #3392 :

> these 2 assertions are not in the scope of this test

Reply:
Why not? We want to make sure the role and sortOrder is not leaked if it's delegated right?

Bob shares his inbox to alice, when alice retrieves the shared mailbox, she should not be able to see its role (so should be empty) neither the sortOrder. How are those two assertions not wanted?

Reply:
in fact it's not clear to me what is this test intent, do we want to show that we can retrieve this shared mailbox or we want to ensure role should be handle in a specific way?

Reply:
That role should be handled in a specific way. The retrieve this shared mailbox case is largely represented in other cases I think. So do you mean I should just test the `role` and `sortOrder` ?

Reply:
The name of the test btw is `getMailboxesShouldReturnSharedMailboxesWithoutRole`... I thought it made the purpose of the test clear? Like shared mailboxes shouldn't not have their roile exposed to sharees?